.github/sandbox_workflows.md

```
@@ -0,0 +1,251 @@
+# Documentation for GitHub Actions Pipelines: Sandbox Creation, Sandbox Deletion and Website Deploy
+
+This documentation provides an overview of three GitHub Actions workflows used for managing AWS CodePipeline executions: Sandbox Creation, Sandbox Deletion, and Website Deploy. It includes instructions on the required secrets and how to add them to your GitHub repository.
+This documentation provides an overview of two GitHub Actions workflows used for managing AWS CodePipeline executions: Sandbox Creation and Sandbox Deletion. It includes instructions on the required secrets and how to add them to your GitHub repository.
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Workflow 1: Sandbox Creation](#workflow-1-sandbox-creation)
+   - [Creation Overview](#creation-overview)
```

Markdownlint (0.35.0)

Expected: 2; Actual: 3
Unordered list indentation

(MD007, ul-indent)